packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -1298,6 +1298,11 @@ export function createLodestarMetrics(
         name: "lodestar_precompute_next_epoch_transition_waste_total",
         help: "Total number of precomputing next epoch transition wasted",
       }),
+      duration: register.histogram({
+        name: "lodestar_precompute_next_epoch_transition_duration_seconds",
+        help: "Duration of precomputeNextEpochTransition at lodestar side",
```

Comment:
The "at lodestar side" might be confusing, are we precomputing the next epoch transition somewhere else (not in lodestar)?

Looking at the code the two major tasks we do are
- precompute next epoch transition
- and prepare execution payload

Might be missing something

Reply:
There's also the prepareExecutionPayload call that's not included in this timer.

Reply:
But isn't that a different task or maybe I misunderstand what is meant by `precomputeEpochTransition`, my assumption is this includes epoch transition + hashing the state.

From my understanding the "at lodestar side" is redundant and can be removed as the task `precomputeNextEpochTransition` is only executed by Lodestar.

`prepareExecutionPayload` is part of preparing for next slot but not part of precomputing the epoch transition.